rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize.cpp

```diff
@@ -61,16 +61,16 @@ void ResizeNode::create_node() {
 void ResizeNode::update_node() {
     std::vector<uint32_t> src_h_dims, src_w_dims;
     // Using original width and height instead of the decoded width and height for computing resize dimensions
-    src_w_dims = _inputs[0]->info().get_original_width_vec();
-    src_h_dims = _inputs[0]->info().get_original_height_vec();
+    src_w_dims = _inputs[0]->info().get_orig_roi_width_vec();
```

Comment:
Can we retain the name as get_original_roi_width_vec() since that seems more clear and better name

Reply:
Done